app.py

```diff
@@ -35,10 +35,16 @@
 init_logging()
 prometheus_registry = CollectorRegistry()
 
+_GRAPH_DB = GraphDatabase()
```

Comment:
Initialized at this line, please let me know if I should move it somewhere else

Reply:
Look good! Thanks! :+1: